jib-core/src/integration-test/java/com/google/cloud/tools/jib/api/JibIntegrationTest.java

```diff
@@ -83,7 +83,8 @@ public void testBasic_helloWorld()
                         RegistryImage.named(targetImageReference)
                             .addCredentialRetriever(
                                 () -> Optional.of(Credential.from("username", "password"))))
-                    .setAllowInsecureRegistries(true));
+                    .setAllowInsecureRegistries(true)
+                    .setAlwaysCacheBaseImage(true));
```

Comment:
ditto, could you remove all these from tests, since you don't test anything with it? Let's preserve the current test behavior.

Reply:
Done.